src/main/java/org/myrobotlab/service/JMonkeyEngine.java

```diff
@@ -1431,6 +1444,9 @@ public void onPointCloud(PointCloud pc) {
   public void onRegistered(Service service) throws Exception {
     // new service - see if we can virtualize it
     log.info("{}.onRegistered({})", getName(), service);
+    if (service.getName().contentEquals("i01.head.jaw")) {

```

Comment:
This is some debug logging / test code that should probably be cleaned up/commented out

Reply:
yeah ... pointcloud resolving to jmonkey polygon is way in the future .. 
I need to focus on low level (Arduino & Servo) then move on to InMoov (which will be big) and then JMonkey ... I agree - but consider this low priority and still needing debugging .. meaning anyone wanting to work on it directly can and also clean up the debugging
I'd be glad to do it .. but not there yet - and all this is certainly better that the previous JMonkey App which was completely hardcoded to InMoov (while now it dynamically loads any model you dump into the assets directory - "huge improvement")